phys2bids/interfaces/txt.py

```diff
@@ -14,6 +14,34 @@
 LGR = logging.getLogger(__name__)
 
 
+def multifreq(timeseries, freq):
+    """
+    Checks if there are channels with different frequency than the maximum one
```

Comment:
Could you please comment a bit more on what the function is doing?

Reply:
sure